src/Symfony/Component/Inflector/Inflector.php

```diff
@@ -30,6 +30,9 @@ final class Inflector
         // Fourth entry: Whether the suffix may succeed a consonant
         // Fifth entry: singular suffix, normal
 
+        // podiums (podium)
```

Comment:
IIUC it's generally prefered over podia? https://dictionary.cambridge.org/dictionary/english/podium